src/epcpm/mainwindow.py

```diff
@@ -414,8 +422,10 @@ def open_project(self, filename=None, project=None):
             view.customContextMenuRequested.connect(m)
 
         self.uuid_notifiers["can"].set_view(self.ui.can_view)
-        self.uuid_notifiers["sunspec"].set_view(self.ui.sunspec_view)
+        self.uuid_notifiers["staticmodbus"].set_view(self.ui.static_modbus_view)
```

Comment:
was this supposed to be an addition to the staticmodbus story?

Reply:
I dont see much staticmodbus replacement, looks a bit out of context

Reply:
This is an area of the code I noticed didn't have the static modbus code done correctly. I fixed it while I was in the vicinity. There is another section above this where I removed some static modbus code which was actually invalid. I don't think I've run across that particular bit above in testing because I think it might be dead/orphaned code.

So yes, in general, this is not a static modbus PR, but I did fix a couple things when I saw them.